lib/ccf/structural_preprocessing/one_subject_job_submitter.py

```diff
@@ -8,7 +8,8 @@
 import shutil
 import stat
 import subprocess
-
+import sys
```

Comment:
Most, but not all, of the changes in this file seem to be for the submitting jobs from "local" machine feature.